src/commands/generate/app-icon.ts

```diff
@@ -0,0 +1,44 @@
+import { GluegunToolbox } from "gluegun"
+import { generateAppIcons, runGenerator, validateAppIconGenerator } from "../../tools/generators"
+import { command, heading, p, warning } from "../../tools/pretty"
+
+module.exports = {
+  alias: ["launcher-icon"],
+  description: "Generates app-icons from templates",
+  run: async (toolbox: GluegunToolbox) => {
+    const generator = toolbox.parameters.command?.toLowerCase()
```

Comment:
the `generator` is no longer a parameter, but a command since it's its own file and therefore command.